src/service/fixed-layer.js

```diff
@@ -450,11 +512,14 @@ export class FixedLayer {
         // large value (to catch cases where sticky-tops are in a long way
         // down inside a scroller).
         for (let i = 0; i < elements.length; i++) {
-          setImportantStyles(elements[i].element, {
-            top: '',
-            bottom: '-9999vh',
-            transition: 'none',
-          });
+          const fe = elements[i];
+          if (!fe.independent) {
```

Comment:
Are independent elements even tracked in fixed layer? They don't participate in offsetting or transfer.

Reply:
They're transferred and setup initially, they just don't participate in subsequent transformMutates.

Reply:
Remove them from fixed layer? Then we can avoid the entire interaction. When you want them tracked again, add them back.

Reply:
[They need to be in the fixed layer for iOS.](https://drive.google.com/file/d/0B_v8thsbiGyDMXZMZkRFZGFRbjA/view) Are you suggesting we leave independent elements in the transfer layer but not track them in `FixedLayer.elements_`? 

(For this particular use case, we wouldn't need to re-track)

Reply:
Ok, I understand now. It feels like we could just add them to the transfer layer directly, not track them as `fe`s.

Reply:
Or, can we combine this with the lightbox usecase that @choumx wrote? I need to keep this class simple, it's already too complicated for me to understand.